### PR TITLE
SEO

### PR DESCRIPTION
**Pull request by sarah-arrrgh**

### CODE REVIEW

File: book.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "anchors"
   ],
   "variables": {
-    "org": "Root Systems"
+    "org": "Root Systems",
+    "title": "Enspiral Root Systems NZ",
+    "description": "Enspiral Root Systems are a small high-trust livelihood pod doing tech consulting and software development within the Enspiral network in Wellington. Our purpose is to imagine and cultivate an ecosystem of local, replicable, life-support systems for interdependent abundance."
```
Comment (ahdinosaur) [inline book.json]: "Enspiral Root Systems are" -> "Enspiral Root Systems is"

File: book.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
     "anchors"
   ],
   "variables": {
-    "org": "Root Systems"
+    "org": "Root Systems",
+    "title": "Enspiral Root Systems NZ",
```
Comment (ahdinosaur) [inline book.json]: i thought `title` and `description` were top-level properties in the `book.json`: https://toolchain.gitbook.com/config.html. do these values show up in the `<title></title>` and `<meta name="description"></meta>` tags?

Comment (sarah-arrrgh) [reply book.json]: Ah I gotcha... I moved them to the top level then inspected it running on the server - the <title> shows up in the <head> but not the <description>... unsure why!

**Comment by sarah-arrrgh** [comment]
@ahdinosaur ?

**Comment by ahdinosaur** [comment]
updated https://www.gitbook.com/book/enspiral-root-systems/handbook/settings to match, since it feels like the hosted GitBook version uses this instead of the `book.json` values, although the description was too long to fit so left out the first sentence. looking at the [page html source](view-source:https://www.rootsystems.nz/), it seems like the description meta tag still isn't showing up (it's `<meta name="description" content="">`). maybe we should use something like https://plugins.gitbook.com/plugin/meta?

**Pull request merged**